Demo/Configuration.swift

```diff
@@ -243,11 +243,11 @@ internal struct DemoAppSettings: Codable {
             showsStorePaymentMethodField: cardSettings.showsStorePaymentMethodField,
             showsSecurityCodeField: cardSettings.showsSecurityCodeField,
             koreanAuthenticationMode: cardSettings.koreanAuthenticationMode,
-            socialSecurityNumberMode: cardSettings.socialSecurityNumberMode,
-            storedCardConfiguration: storedCardConfig,
-            installmentConfiguration: installmentConfiguration,
-            billingAddress: billingAddressConfig
+            socialSecurityNumberMode: cardSettings.socialSecurityNumberMode
         )
+        .stored(storedCardConfig)
```

Comment:
There is a discrepancy, in the example above it was `.stored(stored)`
Let's pick one style